package.json

```diff
@@ -247,11 +240,10 @@
         "intl-messageformat-parser": "^3.6.4",
         "jest": "^29.7.0",
         "jest-environment-jsdom": "^29.7.0",
-        "jest-image-snapshot": "^2.12.0",
-        "jest-puppeteer": "^4.4.0",
+        "jest-image-snapshot": "^6.4.0",
```

Comment:
do we even need this anymore? we dont run image snapshots anymore

Reply:
we don't run snapshots but in case we decide to 🤷 

some scripts and the jest image config still reference this so until we remove those